File: Beecrowd/1010 Cálculo Simples/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int codigop1, codigop2, qtd1, qtd2;
	float valor1, valor2, total;

	cin >> codigop1;
    cin >> qtd1;
	cin >> valor1;

	cin >> codigop2;
    cin >> qtd2;
	cin >> valor2;
	
	total = (qtd1 * valor1) + (qtd2 * valor2);
	
	cout << fixed;
	cout.precision(2);
	cout << "VALOR A PAGAR: R$ " << total << endl;
 
    return 0;
}

File: Beecrowd/1010 Cálculo Simples/main.py
entrada = input().split()
codigop1 = int(entrada[0])
qtd1 = int(entrada[1])
valor1 = float(entrada[2])

entrada1 = input().split()
codigop2 = int(entrada1[0])
qtd2 = int(entrada1[1])
valor2 = float(entrada1[2])

total = (qtd1 * valor1) + (qtd2 * valor2)

print("VALOR A PAGAR: R$ %0.2f" %total)